### PR TITLE
[Common Lisp] Using gender neutral pronouns for Lisp Aliens.

### LINKED ISSUE #2855: [Common Lisp] Improve Concept Exercises pronoun usage.
I have noticed that some exercises use gendered pronouns when referring to the Lisp Alien characters. Due to our lack of knowledge of Lisp Alien biology  and society it seems better to use the singlular `they` instead than to presume 'she' or 'he' based upon our English versions of their names.

To correct this issue the existing exercises should be reviewed and have their pronoun usage corrected.

If in the future we learn more clearly about their biology and society the maintainers of this track can choose better pronouns ot use, perhaps translating Lisp Alien's own pronouns (if they use them) into English.

### PR DESCRIPTION
**Pull request by verdammelt**
Given our lack of understanding of Lisp Alien biology and societal norms it is best to use gender neutral pronouns for them for the time being until we get to know them better.

----

Fixes #2855

### CODE REVIEW

File: languages/common-lisp/exercises/concept/leslies-lists/.docs/instructions.md
Comment (TheLostLambda) [inline languages/common-lisp/exercises/concept/leslies-lists/.docs/instructions.md:15]: I think we'll need to change the verb conjugations after many of the swapped pronouns. E.g. "they needs" -> "they need"

Other than that, I think this is a good change :)

**Review by TheLostLambda — CHANGES_REQUESTED** [review]

File: languages/common-lisp/exercises/concept/leslies-lists/.docs/instructions.md
Comment (verdammelt) [reply languages/common-lisp/exercises/concept/leslies-lists/.docs/instructions.md]: @TheLostLambda please take another look I believe I have fixed the verb conjugations now.

File: languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/instructions.md
Comment (TheLostLambda) [inline languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/instructions.md:2]: Looks like we have a sneaky "him" that got left behind here!

Comment (TheLostLambda) [reply languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/instructions.md:2]: And a "he's" on the next line :)

**Review by TheLostLambda — CHANGES_REQUESTED** [review]
Nearly there! Maybe this is everything still kicking around?

```
common-lisp/exercises/concept on  common-lisp/fix-pronoun-usage [$?] 
❯ rg "\bhe\b|\bhis\b|\bhim\b|\bshe\b|\bher\b|\bhers\b" --hidden
pal-picker/.docs/instructions.md
60:Ludwig could use a simple function called `feeding-time-p` to alert him when the

socks-and-sexprs/.docs/instructions.md
2:wants to go hang out with all the other lisp aliens, but their parents have told him
3:that he's not going anywhere until their room is clean.
```

File: languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/instructions.md
Comment (verdammelt) [reply languages/common-lisp/exercises/concept/socks-and-sexprs/.docs/instructions.md:2]: Ye Gods! This is getting embarrassing!

File: languages/common-lisp/exercises/concept/pal-picker/.docs/instructions.md
Comment (TheLostLambda) [inline languages/common-lisp/exercises/concept/pal-picker/.docs/instructions.md:60]: So far as I can tell, the "him" here is the last one! Then I'm happy to merge!

Comment (verdammelt) [reply languages/common-lisp/exercises/concept/pal-picker/.docs/instructions.md]: OK, squashed this one.

This was supposed to be an "easy" issue to fix! I should  have known better :).

Once you approve (assuming you don't find any more issues) I'll merge.

**Review by TheLostLambda — APPROVED** [review]
Looks good to me! Sorry this was so much of a faff! I'll be sure to used gender neutral pronouns from here on out :)

**Pull request merged**
